src/lib/list/list-item.html

```diff
@@ -5,7 +5,13 @@
   </div>
 
   <ng-content
-      select="[md-list-avatar],[md-list-icon], [mat-list-avatar], [mat-list-icon]"></ng-content>
-  <div class="mat-list-text"><ng-content select="[md-line], [mat-line]"></ng-content></div>
+      select="[md-list-avatar], [md-list-icon], [mat-list-avatar], [mat-list-icon],
+              [mdListAvatar], [mdListIcon], [matListAvatar], [matListIcon]">
+  </ng-content>
+
+  <div class="mat-list-text">
```

Comment:
A little more context: The :empty selector is used to set display:none on the mat-list-text div so it doesn't take up space in the flexbox container leaving no room for anything else. The actual text seems to come after that div rather than inside it.

https://github.com/angular/material2/blob/5bc97ec89462a2dc11fdeccf69ee41eadc265bd4/src/lib/list/list.scss#L80
https://github.com/angular/material2/blob/dcc857664c9313694d020845135f1ce63a66f43a/src/lib/core/style/_list-common.scss#L46